packages/kbn-securitysolution-ecs/kibana.jsonc

```diff
@@ -1,5 +1,9 @@
 {
   "type": "shared-common",
   "id": "@kbn/securitysolution-ecs",
-  "owner": "@elastic/security-threat-hunting-explore"
-}
+  "owner": [
+    "@elastic/security-threat-hunting-explore"
+  ],
+  "group": "platform",
```

Comment:
I don't see `securitysolution-ecs` package being used by any other solution. Is this correct?

Reply:
disregard this, I see o11y imports it